File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int x = 0;
        for (auto &operation : operations){
            if (operation[1] == '-')
                --x;
            else if (operation[1] == '+')
                ++x;
        }
        return x;
    }
};

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        
        int x = 0;
        foreach (var operation in operations)
        {
            if (operation == "++X" || operation == "X++")
                ++x;
            else
                --x;
        }
        return x;
    }
}